pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.styles.ts

```diff
@@ -6,16 +6,19 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
   const backgroundColor = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#161719' }, theme.type);
   const borderColor = selectThemeVariant(
     { light: (theme.colors as any).gray85, dark: '#292929' },
-    theme.type
+    theme.type,
   );
   const headerBackground = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#3D3D3D' }, theme.type);
   const textColor = selectThemeVariant(
     { light: (theme.colors as any).gray85, dark: 'rgba(255, 255, 255, 0.8)' },
-    theme.type
+    theme.type,
   );
 
   return {
     /* This will make the table scrollable when it gets too small */
+    scrollableContent: css`
+      padding-bottom: 21px !important;
```

Comment:
Maybe make this padding number a variable from the theme like `theme.spacing.md`

Reply:
@tiagomotasantos we don't need to do it because it's a quite specific value that we're using only in one place